src/pages/index.js

```diff
@@ -64,50 +83,155 @@ profileEditPopup.setEventListeners();
 const imagePreviewPopup = new PopupWithImages("#card-picture-modal");
 imagePreviewPopup.setEventListeners();
 
+const cardDeletePopup = new PopupWithConfirmation("#delete-card-modal");
+cardDeletePopup.setEventListeners();
+
 const userInfo = new UserInfo({
   nameSelector: "#profile-title",
   jobSelector: "#profile-description",
+  pictureSelector: "#profile-image",
+});
+
+api.getUserInfo().then((userData) => {
+  userInfo.setUserInfo({ name: userData.name, about: userData.about });
+  userInfo.setUserAvatar({ avatar: userData.avatar });
 });
 
 //Event listeners
 
 profileEditButton.addEventListener("click", function () {
-  profileTitleInput.value = profileTitle.textContent.trim();
-  profileDescriptionInput.value = profileDescription.textContent.trim();
-  editFormValidator.resetValidation();
   profileEditPopup.open();
```

Comment:
* You need to **insert** the profile data into the inputs when you open the **profile modal** to let the user **edit** the data and submit it.

But you need to use method `getUserInfo` from `UserInfo` to get the data